app.ts

```diff
@@ -3,11 +3,21 @@ import * as cors from "cors";
 import * as express from "express";
 import * as helmet from "helmet";
 import * as morgan from "morgan";
+import * as xss from "xss-clean";
+import * as rateLimit from "express-rate-limit";
+import * as mongoSanitize from "express-mongo-sanitize";
+import * as hpp from "hpp";
 import * as errorHandler from "./src/helpers/error.handler";
 import apiV1 from "./src/api";
 import config from "./src/config/config";
 import { MongoHelper } from "./src/config/mongodb.config";
 
+// Rate limiting
+const limiter = rateLimit({
```

Comment:
do we really need this package? cant it be handled through the api manager?

Reply:
@sajeetharan I'm not sure whether it can be handled through the API manager or not. if you don't need it, I can remove it and push other files without this package.